content/en/docs/ops/best-practices/security/index.md

```diff
@@ -7,36 +7,170 @@ owner: istio/wg-security-maintainers
+
+* `Gateway`s, which controls the ports and TLS settings for the gateway.
+* `VirtualService`s, which control the routing logic. These are associated with `Gateway`s by direct reference in the `gateways` field and a mutual agreement on the `hosts` field in the `Gateway` and `VirtualService`.
+
+### Restrict `Gateway` creation privileges
+
+It is recommended to restrict creation of Gateway resources to trusted cluster administrators. This can be achieved by [Kubernetes RBAC policies](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) or tools like [Open Policy Agent](https://www.openpolicyagent.org/).
+
+### Avoid overly broad `hosts` configurations
+
+When possible, avoid overly broad `hosts` settings in `Gateway`.
+
+For example, this configuration will allow any `VirtualService` to bind to the `Gateway`, potentially exposing unexpected domains:
+
+{{< text yaml >}}
+servers:
+- port:
+    number: 80
+    name: http
+    protocol: HTTP
+  hosts:
+  - "*"
+{{< /text >}}
+
+This should be locked down to allow only specific domains or specific namespaces:
+
+{{< text yaml >}}
+servers:
+- port:
+    number: 80
+    name: http
+    protocol: HTTP
+  hosts:
+  - "foo.example.com" # Allow only VirtualServices that are for foo.example.com
+  - "default/bar.example.com" # Allow only VirtualServices in the default namespace that are for bar.example.com
+  - "route-namespace/*" # Allow only VirtualServices in the route-namespace namespace for any host
+{{< /text >}}
+
+### Isolate sensitive services
+
+It may be desired to enforce stricter physical isolation for sensitive services. For example, you may want to run a
+[dedicated gateway instance](/docs/setup/install/istioctl/#configure-gateways) for a sensitive `payments.example.com`, while utilizing a single
+shared gateway instance for less sensitive domains like `blog.example.com` and `store.example.com`.
+
+## Protocol detection
+
+Istio will [automatically determine the protocol](/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection) of traffic it sees.
+To avoid accidental or intentional miss detection, which may result in unexpected traffic behavior, it is recommended to [explicitly declare the protocol](/docs/ops/configuration/traffic-management/protocol-selection/#explicit-protocol-selection) where possible.
+
+## CNI
+
+In order to transparently capture all traffic, Istio relies on `iptables` rules configured by the `istio-init` `initContainer`.
+This adds a [requirement](/docs/ops/deployment/requirements/) for the `NET_ADMIN` and `NET_RAW` [capabilities](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container) to be available to the pod.
+
+To reduce privileges granted to pods, Istio offers a [CNI plugin](/docs/setup/additional-setup/cni/) which removes this requirement.
+
+{{< warning >}}
+The Istio CNI plugin is currently an alpha feature.
```

Comment:
@stewartbutler @mandarjog @justinpettit heads up, this will put pressure on carrying istio/cni into beta